util/downloader/err.go

```diff
@@ -0,0 +1,13 @@
+package downloader
```

Comment:
1- Please rename the file to `errors.go` to maintain consistency with the rest of the project.
2- These errors don't carry useful information since they lack a reason for the error.